cmd/sealer/cmd/image/rmi.go

```diff
@@ -22,15 +22,24 @@ import (
 
 var removeOpts *options.RemoveImageOptions
 
+var exampleForRmiCmd = `sealer rmi registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8
+
+prune dangling images:
+        sealer rmi --prune/-p
+
+force removal of the image and any containers using the image:
+        sealer rmi registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.19.8 --force/-f
+
```

Comment:
remove the line.

Reply:
No need for -f flag example?

Reply:
I notice there is a empty line.
I'm not sure if the content is exhibited correctly in my machine.